### PR TITLE
refactor(pie-modal): DSW-000 pie modal page object refactor

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
## Describe your changes (can list changeset entries if preferable)
- Updated pie modal page object to match format in pie cookie banner

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code

### Reviewer 1
- No docs site changes.

### Reviewer 2
- No docs site changes.

### CODE REVIEW

**Review by jamieomaguire — DISMISSED** [review]

File: packages/components/pie-modal/test/component/pie-modal.spec.ts [hunk 1]
```diff
@@ -15,11 +15,11 @@ import {
     headingLevels,
 } from '../../src/defs.ts';
 
-let modalPage: PieModalPage;
+let modalPage: PieModalComponent;
```
Comment (siggerzz) [inline packages/components/pie-modal/test/component/pie-modal.spec.ts:18]: Probably worth updating these to `modalComponent`

**Review by siggerzz — CHANGES_REQUESTED** [review]
Minor comment regarding variables name!

**Review by jamieomaguire — DISMISSED** [review]

File: packages/components/pie-modal/test/component/pie-modal.spec.ts [hunk 2]
```diff
@@ -32,10 +32,10 @@ test.describe('modal', () => {
         });
 
         // Act
-        const modal = await modalPage.isModalVisible();
+        const modal = await modalComponent.isModalVisible();
```
Comment (siggerzz) [inline packages/components/pie-modal/test/component/pie-modal.spec.ts:35]: Could you we change this to `isModalVisible`?

Comment (siggerzz) [reply packages/components/pie-modal/test/component/pie-modal.spec.ts:35]: Or change to `expect(await modalComponent.isModalVisible()).toBe(true);` so it's consistent with other tests.

**Review by siggerzz — CHANGES_REQUESTED** [review]
Few minor comments around naming

**Review by siggerzz — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]
Lovely stuff

**Pull request merged**